pkg/api/transaction_handler/bitcoin_node.go

```diff
@@ -1,4 +1,4 @@
-package transaction_handler
+package transactionhandler
```

Comment:
I think the underscore is valid as the rule has been excluded

Reply:
Came back to the option with underscore